mergify_engine/tests/unit/test_json.py

```diff
@@ -42,6 +44,11 @@ class Colour(enum.Enum):
 mergify_json.register_type(type(Colour.BLUE))
 
 
+def test_register_type_fail():
+    with pytest.raises(RuntimeError):
+        mergify_json.register_type(type(Colour.BLUE))
```

Comment:
Damn I realize this was not fixed L44…
```suggestion
        mergify_json.register_type(Colour)
```